dataplane/src/packet_processor/ingress.rs

```diff
@@ -45,7 +45,12 @@ impl Ingress {
         if packet.try_ipv4().is_some() || packet.try_ipv6().is_some() {
             match &interface.attachment {
                 Some(Attachment::VRF(fibr)) => {
-                    let vrfid = fibr.get_id().unwrap().as_u32();
+                    let Some(vrfid) = fibr.get_id().map(|x| x.as_u32()) else {
+                        /* we may ocassionaly not be able to enter a fib on reconfigs */
+                        warn!("Failed to access fib on ingress!");
```

Comment:
When does this happen and why?

Reply:
It is a sanity to get rid of the unwrap(). It could happen during the process of removing a VRF until we'd detach the interface.